routes/events.js

```diff
@@ -47,4 +56,18 @@ module.exports = function(app) {
             }
         });
     });
+
+    // Route for retrieving data of all events in a tier
+    app.get('/events/:tier', function(req, res, next) {
+
+        eventServices.findAllEventsInTier(function(err, events) {
```

Comment:
how are you finding all of the events in a tier without passing the name of the tier to `findAllEventsInTier`?